flags/eigendaflags/cli.go

```diff
@@ -0,0 +1,121 @@
+package eigendaflags
+
+import (
+	"time"
+
+	"github.com/Layr-Labs/eigenda/api/clients"
+	"github.com/Layr-Labs/eigenda/api/clients/codecs"
+	"github.com/urfave/cli/v2"
+)
+
+// TODO: we should eventually move all of these flags into the eigenda repo
+
+var (
+	DisperserRPCFlagName                 = withFlagPrefix("disperser-rpc")
+	StatusQueryRetryIntervalFlagName     = withFlagPrefix("status-query-retry-interval")
+	StatusQueryTimeoutFlagName           = withFlagPrefix("status-query-timeout")
+	DisableTLSFlagName                   = withFlagPrefix("disable-tls")
+	ResponseTimeoutFlagName              = withFlagPrefix("response-timeout")
+	CustomQuorumIDsFlagName              = withFlagPrefix("custom-quorum-ids")
+	SignerPrivateKeyHexFlagName          = withFlagPrefix("signer-private-key-hex")
+	PutBlobEncodingVersionFlagName       = withFlagPrefix("put-blob-encoding-version")
+	DisablePointVerificationModeFlagName = withFlagPrefix("disable-point-verification-mode")
+	WaitForFinalizationFlagName          = withFlagPrefix("wait-for-finalization")
+)
+
+func withFlagPrefix(s string) string {
+	return "eigenda." + s
+}
+
+func withEnvPrefix(envPrefix, s string) []string {
+	return []string{envPrefix + "_EIGENDA_" + s}
+}
+
+// CLIFlags ... used for EigenDA client configuration
+func CLIFlags(envPrefix, category string) []cli.Flag {
+	return []cli.Flag{
+		&cli.StringFlag{
+			Name:     DisperserRPCFlagName,
+			Usage:    "RPC endpoint of the EigenDA disperser.",
+			EnvVars:  withEnvPrefix(envPrefix, "DISPERSER_RPC"),
+			Category: category,
+		},
+		&cli.DurationFlag{
+			Name:     StatusQueryTimeoutFlagName,
+			Usage:    "Duration to wait for a blob to finalize after being sent for dispersal. Default is 30 minutes.",
```

Comment:
Knit - hasn't this value been changed given the recent decrease in bridging frequency?

Reply:
That's a good question. Do right now bridging frequency is:
- preprod: 1 min
- testnet: 10 min (still lol... but guessing they'll change it to 3 min also eventually..?)
- mainnet: 3 min

Finalization itself requires waiting for 24 minutes though. So not even sure how 30 minutes was enough.
Feel like the msg in this flag should also reflect whether the WaitForFinalization flag is true or false?